frontend/www/js/omegaup/components/common/Navbarv2.vue

```diff
@@ -1,280 +1,290 @@
 <template>
-  <div class="nav-container fixed-top">
-    <nav class="navbar navbar-expand-lg navbar-light">
-      <a class="navbar-brand" href="/">
-        <img alt="omegaUp" src="/media/omegaup_curves.png" />
-        <img
-          alt="lockdown"
-          title="lockdown"
-          v-bind:src="lockDownImage"
-          v-show="omegaUpLockDown"
-        />
-      </a>
-      <button
-        class="navbar-toggler"
-        type="button"
-        data-toggle="collapse"
-        data-target=".omegaup-navbar"
-        aria-expanded="false"
-        aria-label="Toggle navigation"
-      >
-        <span class="navbar-toggler-icon"></span>
-      </button>
-      <div class="collapse navbar-collapse omegaup-navbar">
-        <ul class="navbar-nav mr-auto" v-if="!omegaUpLockDown && !inContest">
-          <li
-            class="nav-item"
-            v-bind:class="{ active: navbarSection === 'arena' }"
-          >
-            <a class="nav-link" href="/arena/">{{ T.navArena }}</a>
-          </li>
-          <li
-            class="nav-item dropdown"
-            v-bind:class="{ active: navbarSection === 'contests' }"
-            v-if="isLoggedIn && isMainUserIdentity"
-          >
-            <a
-              class="nav-link dropdown-toggle"
-              href="#"
-              role="button"
-              data-toggle="dropdown"
-              aria-haspopup="true"
-              aria-expanded="false"
+  <header class="nav-container fixed-top">
+    <div class="container-xl">
```

Comment:
```suggestion
    <div class="container">
```

pregunta: por qué `container-xl` en vez de `container`?

Reply:
![image](https://user-images.githubusercontent.com/20785082/78506702-e702d400-7751-11ea-8a43-1b77532d5fb5.png)

Porque `container` siempre centra el contenido dentro de la página, eso nos da unos márgenes y paddings innecesarios en tamaños pequeños.

![container](https://user-images.githubusercontent.com/20785082/78506749-2df0c980-7752-11ea-979f-8d9197d292a2.gif)